Its.Log/LogEntry.cs

```diff
@@ -366,7 +366,7 @@ internal IEnumerable<object> Confirmations
         }
 
         [FormatterSkipsOnNull]
-        internal IEnumerable<object> Extensions
+        public IEnumerable<object> Extensions
```

Comment:
Why make this public?

Reply:
If I keep this as internal, in LogEventsTests line 82 I will see error 'LogEntry.Extensions' is inaccessible due to its protection level.

It used to work before but I don't understand why.

Reply:
I can't link LogEntry.cs in Its.Log.UnitTests because I will be defining LogEntry twice (once in my linked LogEntry.cs, once in my imported Its.Log project).